demos/node-classification-graphsage/graphsage-cora-example.py

```diff
@@ -99,8 +99,12 @@ def train(
         test_nodes, test_targets, train_size=500, test_size=None, random_state=523214
     )
 
+    # Create graph schema
+    sample_edges = [e for ii,e in enumerate(G.edges(keys=True)) if ii < 5]
+    schema = G.create_graph_schema(create_type_maps=False, nodes=node_ids[:1], edges=sample_edges)
```

Comment:
why do we need to create the graph schema and pass it to the node generator? This extra step was not there before, what is the justification for putting it in the demo, making the workflow more complicated?

Reply:
You're right .. this shouldn't be there. It was test code that somehow got checked in. I'll remove it.